### DISCUSSION TITLE
各式各樣的深度學習模型

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#347] [Ideas]
最近聽到有人講到一種深度學習的模型，keras的官方連結如下。
[Masked Image Modeling with Autoencoders](https://keras.io/examples/vision/masked_image_modeling/)
我還沒搞懂這個深度學習模型在做什麼。
